### DISCUSSION TITLE
Need to add button in every datarow to work with datarow and datacell

### DISCUSSION THREAD

**Discussion by pamraman2009** [flet-dev/flet#1887] [Q&A]
**Please Describe The Problem To Be Solved**

(Replace This Text: Please present a concise description of the problem to be addressed by this feature request. Please be clear what parts of the problem are considered to be in-scope and out-of-scope.)

**(Optional): Suggest A Solution**

(Replace This Text: A concise description of your preferred solution. Things to address include:
* Details of the technical implementation, e.g. Flutter widget solving the problem.
* Tradeoffs made in design decisions
* Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Comment by ndonkoHenri** [comment]
can you please **Describe The Problem To Be Solved**?
